bokeh/models/callbacks.py

```diff
@@ -24,6 +27,29 @@ class OpenURL(Callback):
 
 class CustomJS(Callback):
     """ Execute a JavaScript function. """
+    
+    def __init__(self, func=None, **kwargs):
+        if func is not None:
+            if not isinstance(func, FunctionType):
+                raise ValueError('CustomJS needs function object, or "args" and "code" properties.')
+            try:
+                from flexx.pyscript import py2js
+            except ImportError:
+                if sys.version_info < (3, ):
+                    raise RuntimeError('Using Python functions for CustomJS '
+                                       'is currently not supported on Python 2.x)')
+                else:
+                    raise RuntimeError('To use Python functions for CustomJS, '
+                                       'you need Flexx (pip install flexx)')
+            # Collect default values
+            default_values = func.__defaults__  # Python 2.6+
+            default_names = func.__code__.co_varnames[:len(default_values)]
+            kwargs['args'] = dict(zip(default_names, default_values))
+            # Get JS code, we could rip out the function def, or just
+            # call the function. We do the latter.
+            kwargs['code'] = py2js(func, 'cb') + 'cb(%s);\n' % ', '.join(default_names)
+
+        Callback.__init__(self, **kwargs)
```

Comment:
Going further, I think `kwargs` shouldn't be overridden silently when user specified `code` and/or `args`.